src/Moryx.Runtime.Maintenance.Web.UI/src/modules/container/ModuleConsole.tsx

```diff
@@ -85,11 +86,29 @@ class ModuleConsole extends React.Component<ModuleConsolePropModel & ModuleConso
 
             let resultEntry = data;
 
-            // Result of void functions is an empty entry
-            if (Object.keys(resultEntry).length > 0) {
-                Config.patchParent(resultEntry, null);
-            } else {
-                resultEntry = new Entry();
+            switch (resultEntry.Value.Type) {
+                case EntryValueType.Byte:
+                case EntryValueType.Boolean:
+                case EntryValueType.Int16:
+                case EntryValueType.UInt16:
+                case EntryValueType.Int32:
+                case EntryValueType.UInt32:
+                case EntryValueType.Int64:
+                case EntryValueType.UInt64:
+                case EntryValueType.Single:
+                case EntryValueType.Double:
+                case EntryValueType.String:
+                case EntryValueType.Enum:
+                case EntryValueType.Collection:
```

Comment:
I am not so familiar with the web implementation, but do you need to patch the parent for collections. Shouldn't the returned entry work as a parent and directly display all entries from the collection? Last question, otherwise thanks for fixing this. 👍

Reply:
Parent patching is only necessary if the entry has sub entries. In the case of collections the sub entries list should be empty?

Reply:
No, in case of collections the sub-entries **are** the collections items. So it needs parent patching?

Reply:
Yes, in that case it needs parent patching. I changed that but I could not test that because the test function

```C#
[EditorBrowsable, Description("Returns the string that was sent.")]
public string[] Echo3(string message)
{
  return new [] { "t1", "t2", "t3"};
}
```

gave me a `StackOverflowException`:

![image](https://user-images.githubusercontent.com/26304706/91485458-2f5e9200-e8ab-11ea-9bfb-20fbc2fac10c.png)

Reply:
Okay, seems to be a problem for arrays of primitives. Thanks for the fix anyways.

Reply:
I created a ticket for this exception
https://github.com/PHOENIXCONTACT/MORYX-Platform/issues/38

Reply:
Fixed by PHOENIXCONTACT/MORYX-Platform#41